cmd/minikube/cmd/start.go

```diff
@@ -77,6 +77,16 @@ var (
 	apiServerNames   []string
 	apiServerIPs     []net.IP
 	hostRe           = regexp.MustCompile(`^[^-][\w\.-]+$`)
+	slowK8sVersions  = map[string]struct{}{
+		"1.18.16": {},
```

Comment:
how did we get this list of versions,? is there a comment or a Url we could add for comments ?

Reply:
I started with 1.20.3, which is the version we know started this issue. 1.18.16 and 1.19.8 are its backports, then I just moved forward in time until present day, testing each release version manually to verify the issue was still present. There's not URL (other than the Kubernetes release page) to add here.